### PR TITLE
asc 4 - added routing and map page

### PR DESCRIPTION
**Pull request by avivt10**

### CODE REVIEW

File: client_react/src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import "./App.css";
-import HomePage from "../src/components/HomePage/homePage";
+import HomePage from "../src/components/HomePage/HomePage";
```
Comment (Shaybush) [inline client_react/src/App.jsx:2]: 1. why did you change that ?
2. use just one /homePage

File: client_react/src/AppRoutes.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React, { Suspense, useState } from "react";
+import { Route, BrowserRouter as Router, Routes } from "react-router-dom";
+import _Loader from "./components/Loader/_Loader";
```
Comment (Shaybush) [inline client_react/src/AppRoutes.jsx:3]: use single call /loader with lowerase

File: client_react/src/AppRoutes.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+const AppRoutes = () => {
+    const Login = React.lazy(() => import("./components/HomePage/HomePage"));
+    const Chat = React.lazy(() => import("./components/HomePage/HomePage"));
```
Comment (Shaybush) [inline client_react/src/AppRoutes.jsx:7]: why home page declare twice?

File: client_react/src/components/DashboardPage/Dashboard.jsx [hunk 4]
```diff
@@ -3,25 +3,40 @@ import DropdownButton from 'react-bootstrap/esm/DropdownButton';
...
+import mapIcon from "../../assets/icons/map.png"
+import { useNavigate } from "react-router-dom";
 
 const Dashboard = ({ existUser, setExistUser, setIsLogin }) => {
+  const navigate = useNavigate();
   // log out function to log the user out of google and set the profile array to null
   const logOut = () => {
     googleLogout();
-    setExistUser(null);
+    existUser = null;
```
Comment (Shaybush) [inline client_react/src/components/DashboardPage/Dashboard.jsx:14]: use set

File: client_react/src/components/DashboardPage/Dashboard.jsx [hunk 5]
```diff
@@ -3,25 +3,40 @@ import DropdownButton from 'react-bootstrap/esm/DropdownButton';
...
     setIsLogin(false);
+    navigate("/")
   };
 
+  
+  // if(existUser === undefined){
+  //   existUser = JSON.parse(localStorage.getItem("userData"))
+  //   console.log(existUser)
+  //    setIsLogin(false);
+  // }
```
Comment (Shaybush) [inline client_react/src/components/DashboardPage/Dashboard.jsx:25]: delete

**Review by Shaybush — CHANGES_REQUESTED** [review]
fix please

File: client_react/src/components/DashboardPage/Dashboard.jsx (hunk 4 above)
Comment (avivt10) [reply client_react/src/components/DashboardPage/Dashboard.jsx:14]: Done.

**Review by Shaybush — APPROVED** [review]

**Pull request merged**